src/pynwb/testing/mock/ecephys.py

```diff
@@ -69,9 +69,9 @@ def mock_electrodes(
 def mock_ElectricalSeries(
     name: Optional[str] = None,
     description: str = "description",
-    data=None,
-    rate: float = 30000.0,
-    timestamps=None,
+    data: Optional[np.ndarray] = None,
+    rate: Optional[float] = None,
+    timestamps: Optional[np.ndarray] = None,
```

Comment:
```suggestion
    data = None,
    rate: Optional[float] = None,
    timestamps = None,
```

I suggest not adding typing for data and timestamps here since these `ElectricalSeries` arguments can be other array data types (h5py Dataset, Zarr array, TimeSeries, etc.) and so it might be misleading to specify only numpy arrays for the `mock_ElectricalSeries`.

Reply:
Makes sense.